Source/Core/Common/Network.cpp

```diff
@@ -18,6 +18,15 @@
 
 #include <fmt/format.h>
 
+#ifdef __linux__     // features.h is only available on Linux
+#define _GNU_SOURCE  // We have to define this to use macros in features.h
+#include <features.h>
+#ifndef __USE_GNU
+#define __NO_GLIBC__
+#endif
+#undef _GNU_SOURCE
```

Comment:
Won't this cause issues if `_GNU_SOURCE` was already defined?

Reply:
That's true, maybe I can add conditions to define if it's not already defined, and undefine if it's already defined. Would that be good?

Reply:
I think that should do it.

@JosJuice 
IIRC, you have already addressed similar errors with `strerror_r`. Do you know if there are cleaner ways to do it?

Reply:
Sorry, I don't remember any of the details from when I did that work.